### PR TITLE
Allow domain names in egress network policy

### PR DESCRIPTION
**Pull request by pravisankar**
Origin PR: https://github.com/openshift/origin/pull/13002

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/team-documentation PTAL

File: admin_guide/managing_pods.adoc [hunk 1]
```diff
@@ -382,15 +388,21 @@ To configure pod access limits:
...
-first one that matches takes effect. If the two rules in the above example
-were swapped, then traffic would not be allowed to `1.2.3.0/24` because the
-`0.0.0.0/0` rule would be checked first, and it would match and deny all
-traffic.
+first one that matches takes effect. If the three rules in the above example
+were reversed, then traffic would not be allowed to `1.2.3.0/24` and
+`www.foo.com` because the `0.0.0.0/0` rule would be checked first,
+and it would match and deny all traffic.
++
+Domain name updates are reflected within 30 mins. In the above example, lets
```
Comment (ahardin-rh) [inline admin_guide/managing_pods.adoc]: s/mins/minutes
s/lets say/suppose

Comment (ahardin-rh) [inline admin_guide/managing_pods.adoc]: `10.11.12.13`, but later it was changed to `20.21.22.23`. Then, {product-title} will take up to 30 minutes to adapt to these
DNS updates.

**Comment by pravisankar** [comment]
@ahardin-rh updated

**Comment by ahardin-rh** [comment]
Thank you!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/managing_pods.adoc#admin-guide-manage-pods[Managing Pods]
|Added information about allowing domain names in `EgressNetworkPolicy`.
%
|xref:../rest_api/openshift_v1.adoc#rest-api-openshift-v1[{product-title} v1 REST API]
|Added information about allowing domain names in `EgressNetworkPolicy`.
%

**Comment by pravisankar** [comment]
@ahardin-rh fyi, this is targeted for 3.6 release (not 3.5)

**Pull request merged**